{{cookiecutter.profile_name}}/slurm-submit.py

```diff
@@ -109,12 +110,19 @@
     if "{{cookiecutter.account}}" != "":
         arg_dict["account"] = "{{cookiecutter.account}}"
 
+# Ensure output folder for Slurm log files exist.
+# This is a bit hacky; will run for every Slurm submission...
+if "output" in arg_dict:
```

Comment:
arg_dict is initialized further up in the code, thus, output is always present, but initialized to None. The os.makedirs command therefore fails the tests. Could you change to `if arg_dict.get("output", None) is not None:` and similarly for error?

Reply:
Oh, sorry about that! I wasn't thinking straight here. Usually just use the `Namespace` object directly from the `ArgumentParser`, and then I usually just do `if args.output`, which automatically takes care of that edge case. 
I could  change it to simply `if arg_dict["output"]`, but maybe the intent is more clear to follow your suggestion to explicitly check for `not Null`. What would you prefer?